fvm/evm/types/codeFinder.go

```diff
@@ -8,22 +8,6 @@ import (
 )
 
 func ValidationErrorCode(err error) ErrorCode {
-	// if is a EVM validation error first unwrap one level
-	if IsEVMValidationError(err) {
-		// check local errors
-		switch err {
-		case ErrInvalidBalance:
-			return ValidationErrCodeInvalidBalance
-		case ErrInsufficientComputation:
-			return ValidationErrCodeInsufficientComputation
-		case ErrUnAuthroizedMethodCall:
-			return ValidationErrCodeUnAuthroizedMethodCall
-		case ErrWithdrawBalanceRounding:
-			return ValidationErrCodeWithdrawBalanceRounding
-		}
-		err = errors.Unwrap(err)
-	}
-
```

Comment:
These are not part of validation errors anymore, these all requires the flow transaction to revert